src/Jackalope/Transport/DoctrineDBAL/XmlPropsRemover/XmlPropsRemover.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Jackalope\Transport\DoctrineDBAL\XmlPropsRemover;
+
+/**
+ * @internal
+ */
+class XmlPropsRemover
```

Comment:
modernized for PHP 8. but the problems seems not to come from this file - i copied the version from 1.x into my local installation and ran the test and it still fails.